app/src/test/java/com/duckduckgo/app/pixels/EnqueuedPixelWorkerTest.kt

```diff
@@ -122,12 +122,18 @@ class EnqueuedPixelWorkerTest {
         whenever(customTabDetector.isCustomTab()).thenReturn(true)
         whenever(unsentForgetAllPixelStore.pendingPixelCountClearData).thenReturn(1)
         whenever(webViewVersionProvider.getMajorVersion()).thenReturn("91")
-        whenever(defaultBrowserDetector.isDefaultBrowser()).thenReturn(false)
+        whenever(defaultBrowserDetector.isDefaultBrowser()).thenReturn(true)
```

Comment:
Changed this only to reflect the reality, where the browser is default in order for the Custom Tab to make sense.